subworkflows/local/vcf_annotation/main.nf

```diff
@@ -34,7 +34,13 @@ workflow VCF_ANNOTATION {
         ch_vep_extra_files
     )
 
+    // Index VCF file
+
+    TABIX_VEP ( VEP.out.vcf )
+
     emit:
     vcfs = VEP.out.vcf
-    versions = VEP.out.versions
+    tbi = TABIX_VEP.out.tbi
+    versionsvep = VEP.out.versions
+    versionstbi = TABIX_VEP.out.versions
```

Comment:
It might be easier to merge these in the subworkflow to keep the amount of output channels at a minimum

Reply:
versions were merged